docs/installation.rst

```diff
@@ -381,40 +381,38 @@ Continuous Integration Targets
-+----------------------------------+-------------------------------+-----------------------+
-| Ubuntu Linux 16.04 LTS           | 2.7, 3.5, 3.6, 3.7, 3.8,      |x86-64                 |
-|                                  | PyPy, PyPy3                   |                       |
-+----------------------------------+-------------------------------+-----------------------+
-| Windows Server 2012 R2           | 2.7, 3.5, 3.6, 3.7, 3.8       |x86, x86-64            |
-|                                  +-------------------------------+-----------------------+
-|                                  | PyPy, 3.7/MinGW               |x86                    |
-+----------------------------------+-------------------------------+-----------------------+
-
-\* macOS CI is not run for every commit, but is run for every release.
++----------------------------------+--------------------------+-----------------------+
+|**Operating system**              |**Tested Python versions**|**Tested architecture**|
++----------------------------------+--------------------------+-----------------------+
+| Alpine                           | 3.7                      |x86-64                 |
++----------------------------------+--------------------------+-----------------------+
+| Arch                             | 3.7                      |x86-64                 |
++----------------------------------+--------------------------+-----------------------+
+| Amazon Linux 1                   | 3.6                      |x86-64                 |
++----------------------------------+--------------------------+-----------------------+
+| Amazon Linux 2                   | 3.6                      |x86-64                 |
++----------------------------------+--------------------------+-----------------------+
+| CentOS 6                         | 3.6                      |x86-64                 |
++----------------------------------+--------------------------+-----------------------+
+| CentOS 7                         | 3.6                      |x86-64                 |
++----------------------------------+--------------------------+-----------------------+
+| Debian 9 Stretch                 | 3.5                      |x86                    |
++----------------------------------+--------------------------+-----------------------+
+| Debian 10 Buster                 | 3.7                      |x86                    |
++----------------------------------+--------------------------+-----------------------+
+| Fedora 30                        | 3.7                      |x86-64                 |
++----------------------------------+--------------------------+-----------------------+
+| Fedora 31                        | 3.7                      |x86-64                 |
++----------------------------------+--------------------------+-----------------------+
+| macOS 10.14 Catalina             | 3.5, 3.6, 3.7, 3.8       |x86-64                 |
```

Comment:
I would look at GHA and think it's still running 10.13?

Also, 10.14 is Mojave, 10.15 is Catalina.

Reply:
Hmm, the [docs](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources) say `macos-latest` is macOS Catalina 10.15 but checking the logs of the [most recent run](https://github.com/python-pillow/Pillow/commit/5283538fe433de3c0e6855889bf821d19869cdee/checks?check_suite_id=344362814):

```console
$ grep -ril "macOS.*10.13" .
./10_macOS-latest Python 3.5.txt
./macOS-latest Python 3.5/9_Test.txt
./macOS-latest Python 3.5/8_Build.txt
./macOS-latest Python 3.5/12_After success.txt
./7_macOS-latest Python 3.8.txt
./8_macOS-latest Python 3.7.txt
./macOS-latest Python 3.7/9_Test.txt
./macOS-latest Python 3.7/8_Build.txt
./macOS-latest Python 3.7/7_Install macOS dependencies.txt
./macOS-latest Python 3.7/12_After success.txt
./macOS-latest Python 3.8/9_Test.txt
./macOS-latest Python 3.8/11_Docs.txt
./macOS-latest Python 3.8/8_Build.txt
./macOS-latest Python 3.8/7_Install macOS dependencies.txt
./macOS-latest Python 3.8/12_After success.txt
./macOS-latest Python 3.6/9_Test.txt
./macOS-latest Python 3.6/8_Build.txt
./macOS-latest Python 3.6/7_Install macOS dependencies.txt
./macOS-latest Python 3.6/12_After success.txt
./9_macOS-latest Python 3.6.txt
```

```console
$ grep -ril "macOS.*10.14" .
```

```console
$ grep -ril "macOS.*10.15" .
./6_macOS-latest Python pypy3.txt
./macOS-latest Python pypy3/9_Test.txt
./macOS-latest Python pypy3/8_Build.txt
./macOS-latest Python pypy3/12_After success.txt
```
I'll update to match this.

Reply:
Updated in https://github.com/python-pillow/Pillow/pull/4228/commits/578d9ec0165c49ad6a01f4c87fd179a284d03ff1!

Reply:
And reported here: https://github.com/actions/virtual-environments/issues/81#issuecomment-562761489.

Reply:
Thanks